GCEWindowsAgent/accounts.go

```diff
@@ -215,15 +215,15 @@ func printCreds(creds *credsJSON) error {
 
 var badReg []string
 
-func compareAccounts(newKeys windowsKeys, oldStrKeys []string) windowsKeys {
+func compareAccounts(newKeys WindowsKeys, oldStrKeys []string) WindowsKeys {
```

Comment:
Pretty sure the public structs were reverted in the other PR right?

Reply:
yeah i remembered that last night, reverted here now too